src/libpmemobj/lane.h

```diff
@@ -63,34 +61,22 @@ extern "C" {
 
 #define RLANE_DEFAULT 0
 
-enum lane_section_type {
-	LANE_SECTION_ALLOCATOR,
-	LANE_SECTION_LIST,
-	LANE_SECTION_TRANSACTION,
-
-	MAX_LANE_SECTION,
-
-	LANE_ID = MAX_LANE_SECTION
-};
-
-struct lane_section_layout {
-	unsigned char data[LANE_SECTION_LEN];
-};
-
-struct lane_section {
-	/* persistent */
-	struct lane_section_layout *layout;
-
-	void *runtime;
-};
+#define LANE_UNDO_SIZE 2048
+#define LANE_REDO_EXTERNAL_SIZE 640
+#define LANE_REDO_INTERNAL_SIZE 192
+#define LANE_TOTAL_SIZE 3072
 
 struct lane_layout {
-	struct lane_section_layout sections[MAX_LANE_SECTION];
+	struct ULOG(LANE_REDO_EXTERNAL_SIZE) external;
+	struct ULOG(LANE_REDO_INTERNAL_SIZE) internal;
+	struct ULOG(LANE_UNDO_SIZE) undo;
```

Comment:
Please add comments describing purpose of each ULOG.

Reply:
Done